### DISCUSSION TITLE
SO2.4 Bootable USB

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13015] [Unsupported Versions]
Hello,

I'm experiencing an issue with iDrac 8 where a USB created with the SO2.4 ISO using balenaEtcher is not recognized as bootable in UEFI mode. The same USB works fine on iDrac 9 and an Oracle ISO boots correctly on iDrac 8 in UEFI mode.

**Comment by dougburks** [comment]
You might check to see if there is a firmware update available for the iDrac.

**Reply by SliuzasLukas** [reply]
Hello,

iDrac is up-to-date on firmware and any other updates.
